apps/core/factories.py

```diff
@@ -49,3 +49,85 @@ class OmicsUnitFactory(DjangoModelFactory):
     class Meta:
         model = 'core.OmicsUnit'
         django_get_or_create = ('reference', 'strain')
+
+
+class PixelerFactory(DjangoModelFactory):
+
+    date_joined = Faker('date_object')
```

Comment:
It should be a `datetime_*` (see [Faker docs](https://faker.readthedocs.io/en/master/providers/faker.providers.date_time.html)). To check `AbstractUser` fields type, take a look at Django's source code:
https://github.com/django/django/blob/master/django/contrib/auth/models.py#L323